presto-sheets/pom.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.prestosql</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>316-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-sheets</artifactId>
+    <description>Presto - Sheets Readonly Connector</description>
```

Comment:
Presto - Google Sheets Connector?

I do not think there is a point to emphasise that it is read only.